litestar/_openapi/schema_generation/constrained_fields.py

```diff
@@ -80,7 +80,7 @@ def create_string_constrained_field_schema(
         schema.max_length = kwarg_definition.max_length
     if kwarg_definition.pattern:
         schema.pattern = (
-            kwarg_definition.pattern.pattern  # type: ignore[attr-defined]
+            kwarg_definition.pattern.pattern  # type: ignore[attr-defined,unused-ignore]
```

Comment:
```suggestion
            kwarg_definition.pattern.pattern
```

Reply:
For some reason this is raising linter errors locally but not in CI.  I'm wondering if it's a difference between pylance/pyright?